lib/web/ui/integration.go

```diff
@@ -379,6 +379,39 @@ type AWSOIDCListSubnetsResponse struct {
 	NextToken string `json:"nextToken,omitempty"`
 }
 
+// AWSOIDCRequiredVPCSRequest is a request to list VPCs.
+type AWSOIDCListVPCsRequest struct {
+	// Region is the AWS Region.
+	Region string `json:"region"`
+	// AccountID is the AWS Account ID.
+	AccountID string `json:"accountId"`
+	// NextToken is the token to be used to fetch the next page.
+	// If empty, the first page is fetched.
+	NextToken string `json:"nextToken"`
+}
+
+// DatabaseEnrollmentVPC is a wrapper around [awsoidc.VPC] that also includes
+// a link to the ECS service for a deployed Teleport database service in that
+// VPC, if one exists.
+type DatabaseEnrollmentVPC struct {
+	awsoidc.VPC
+	// ECSServiceDashboardURL is a link to the ECS service deployed for this
+	// VPC, if one exists. Can be empty.
+	ECSServiceDashboardURL string `json:"ecsServiceDashboardURL"`
+}
+
+// AWSOIDCDatabaseVPCsResponse contains a list of VPCs, including a link to
+// an existing db service deployment if one exists, and a next token if more
+// pages are available.
+type AWSOIDCDatabaseVPCsResponse struct {
+	// VPCs contains a page of VPCs.
+	VPCs []DatabaseEnrollmentVPC `json:"vpcs"`
+
+	// NextToken is used for pagination.
+	// If non-empty, it can be used to request the next page.
+	NextToken string `json:"nextToken,omitempty"`
```

Comment:
do you know what the default value of a page is? i tried looking it up, and I don't think we set a upper limit, so i'm thinking the default might be 5... could we bump it to 100 or something higher? this way it's highly unlikely we'll require coming back here for next page

Reply:
Hm, I guess it doesn't say what he default is. https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeVpcs.html just says it must be between 5 and 1000 items.

I'll set it to 100  👍